srcpkgs/mesa/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'mesa'
 pkgname=mesa
-version=22.1.7
+version=22.2.2
```

Comment:
22.2.4 is available

Reply:
This has happened a few times. The new versions come and then this won't get merged because it's not tested and then it keeps recycling.

Reply:
patch releases only fix bugs, we should use the latest version available

Reply:
the changes in .3 and .4 are small. test it for a little bit and i'll merge

Reply:
Will update locally too and test gaming, browsing etc. I would normally do on my system

Reply:
Tested with media in mpv, pale moon, some games, seems ok'ish here.. (x86_64-glib - radeonsi)